airflow/api_connexion/openapi/v1.yaml

```diff
@@ -160,6 +160,14 @@ info:
 
     For details on configuring the authentication, see
     [API Authorization](https://airflow.readthedocs.io/en/latest/security/api.html).
+
+    # Update Mask
+    Update mask is available as a parameter in patch endpoints. It is used to notify the
```

Comment:
I am afraid this has little effect on performance as these are only trivial memory operations.

The main reason for this is that you can get the full object with GET, update the selected fields, then send the full object and only indicate the fields to be updated.  In other words, it's necessary to perform a partial update.
```
resource = request.get('/resource/my-id').json()
resource['my-field'] = 'new-value'
request.patch('/resource/my-id?update-mask=my-field', data=json.dumps(resource)) 
```
This makes it easier for the client to use the API.